tests/e2e/controller_test.go

```diff
@@ -83,8 +92,8 @@ type testContextConfig struct {
 
 	operatorControllerTest bool
 	webhookTest            bool
-	components             arrayFlags
-	services               arrayFlags
+	components             TestGroup
+	services               TestGroup
```

Comment:
we are not supposed to extend it, right? It looks as a candidate to to generic TestGroup[]

Reply:
yeah, I was thinking to add a method to the `TestGroup` to bind its own set of flags, so it can be made generic, but it did look ugly, I's think about something as a follow-up